src/test/java/edu/kis/vh/nursery/RhymersJUnitTest.java

```diff
@@ -2,11 +2,12 @@
 
 import org.junit.Assert;
 import org.junit.Test;
+import edu.kis.vh.nursery.stack.IntArrayStack;;
 
 public class RhymersJUnitTest {
 
 	private static final int TEST_VALUE = 4;
-	private static final int EMPTY_STACK_VALUE = -1;
+	private static final int EMPTY_STACK_VALUE = 0;

```

Comment:
The original constant should be used (it would be best to use a common constant in a stacks interface).